moda-contracts/src/Catalog.sol

```diff
@@ -50,7 +50,7 @@ contract Catalog is ICatalog, AccessControlUpgradeable {
     error ReleasesContractIsAlreadyRegistered();
     error ReleasesContractDoesNotHavePermission();
     error ReleaseAlreadyCreated();
-    error UserMustBeMember();
+    error MembershipRequired();
```

Comment:
Membership may be held by a contract, not only an EOA. I'm just simplifying the language to make it less restrictive